src/main/java/org/broadinstitute/dsde/consent/ontology/util/JsonSchemaUtil.java

```diff
@@ -62,12 +84,32 @@ public List<String> validateDataUseV3Schema(String dataUseV3Instance) {
     try {
       ObjectMapper mapper = new ObjectMapper();
       JsonNode jsonSubject = mapper.readTree(dataUseV3Instance);
-      JsonSchema schema = getDataUseInstance();
+      JsonSchema schema = getDataUseV3Instance();
+      Set<ValidationMessage> messages = schema.validate(jsonSubject);
+      return messages.stream().map(ValidationMessage::getMessage).toList();
+    } catch (Exception e) {
+      logError("Unable to load the data use schema: " + e.getMessage());
+      throw new BadRequestException("Invalid schema");
+    }
+  }
+
+  /**
+   * Compares an instance of a data use object to the data use V4 schema
+   *
+   * @param dataUseV4Instance The string instance of a data use object
+   * @return List of validation messages
+   */
+  public List<String> validateDataUseV4Schema(String dataUseV4Instance) {
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      JsonNode jsonSubject = mapper.readTree(dataUseV4Instance);
+      JsonSchema schema = getDataUseV3Instance();
```

Comment:
```suggestion
      JsonSchema schema = getDataUseV4Instance();
```

I think this needs to be V4

Reply:
Nice catch!